src/fides/api/ops/api/v1/endpoints/privacy_experience_config_endpoints.py

```diff
@@ -0,0 +1,327 @@
+    db: Session,
+    component_type: ComponentType,
+    delivery_mechanism: DeliveryMechanism,
+    regions: List[PrivacyNoticeRegion],
+) -> None:
+    """Validates that the supplied regions would be compatible with the proposed ExperienceConfig
+    prior to making the ExperienceConfig changes and linking the Experiences for those regions.
+    """
+    invalid_regions: List[PrivacyNoticeRegion] = []
+    for region in regions:
+        if config_incompatible_with_region(
+            db, component_type, delivery_mechanism, region
+        ):
+            invalid_regions.append(region)
+
+    if invalid_regions:
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"The following regions would be incompatible with this experience: {human_friendly_list([reg.value for reg in invalid_regions])}.",
+        )
+
+
+@router.post(
+    urls.EXPERIENCE_CONFIG,
+    status_code=HTTP_201_CREATED,
+    response_model=ExperienceConfigCreateOrUpdateResponse,
+    dependencies=[
+        Security(
+            verify_oauth_client,
+            scopes=[
+                scope_registry.PRIVACY_EXPERIENCE_CREATE,
+                PRIVACY_EXPERIENCE_UPDATE,
+            ],
+        )
+    ],
+)
+def experience_config_create(
+    *,
+    db: Session = Depends(deps.get_db),
+    experience_config_data: ExperienceConfigCreate,
+) -> ExperienceConfigCreateOrUpdateResponse:
+    """
+    Create Experience Config and then attempt to upsert Experiences and link to ExperienceConfig
+    """
+    experience_config_dict: Dict = experience_config_data.dict(exclude_unset=True)
+    # Pop the regions off the request
+    new_regions: List[PrivacyNoticeRegion] = experience_config_dict.pop("regions")
+
+    # Fail early if the ExperienceConfig is going to be incompatible with any of the regions
+    validate_experience_config_and_region_compatibility(
+        db=db,
+        component_type=experience_config_data.component,
+        delivery_mechanism=experience_config_data.delivery_mechanism,
+        regions=new_regions,
+    )
+
+    logger.info(
+        "Creating experience config of component '{}' and delivery mechanism '{}'.",
+        experience_config_data.component.value,
+        experience_config_data.delivery_mechanism.value,
+    )
+    experience_config = PrivacyExperienceConfig.create(
+        db, data=experience_config_dict, check_name=False
+    )
+
+    if new_regions:
+        logger.info(
+            "Linking regions: {} to experience config '{}'",
+            human_friendly_list([reg.value for reg in new_regions]),
+            experience_config.id,
+        )
+    linked, unlinked, skipped = upsert_privacy_experiences_after_config_update(
+        db, experience_config, new_regions
+    )
+
+    return ExperienceConfigCreateOrUpdateResponse(
+        experience_config=experience_config,
+        linked_regions=linked,
+        unlinked_regions=unlinked,
+        skipped_regions=skipped,
+    )
```

Comment:
This endpoint attempts to _create_ a PrivacyExperienceConfig (with the button labels, component descriptions, etc.) and saves that in one table, and then attempts to _upsert_ PrivacyExperiences for each "region" in the request separately.  

For example, if I am creating a banner overlay PrivacyExperienceConfig and specify the regions are CA, CO, and VA, I make sure banner overlay PrivacyExperiences exist (or create them if they don't) for CA, CO, and VA. I then link each Experience back to the Config via FK.

This is what I'm attempting to surface in the response here with "linked" regions - these are the Experiences that were linked to the Config via FK.  In practice, this endpoint should only have "linked" regions.